### DISCUSSION TITLE
SQL for sessionid store

### DISCUSSION THREAD

**Discussion by denbreaux** [xyhtac/icingatelegram#21] [Polls]
There is an idea to bring permanent storage (i.e. database) for state memory to hold user configuration and sessionid. This will prevent users from requesting access from tg groups again after process restart, but we will need to bring db dependencies and make installation more complicated.

**Comment by xyhtac** [comment]
If we permanently save sessionid we will need to handle its timestamp also, otherwise use access will be granted forever
